[2019-04-30 17:10:51,895] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.6a_change_dir1 2019-04-30T21:10:32.665289+00:00 [queued]>
[2019-04-30 17:10:51,899] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.6a_change_dir1 2019-04-30T21:10:32.665289+00:00 [queued]>
[2019-04-30 17:10:51,899] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:10:51,899] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-04-30 17:10:51,899] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:10:51,907] {__init__.py:1374} INFO - Executing <Task(BashOperator): 6a_change_dir1> on 2019-04-30T21:10:32.665289+00:00
[2019-04-30 17:10:51,908] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Mixed_Failing_Demo', '6a_change_dir1', '2019-04-30T21:10:32.665289+00:00', '--job_id', '231', '--raw', '-sd', 'DAGS_FOLDER/mixed_failing.py', '--cfg_path', '/tmp/tmp4pf5qw7p']
[2019-04-30 17:10:52,164] {base_task_runner.py:101} INFO - Job 231: Subtask 6a_change_dir1 /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-04-30 17:10:52,164] {base_task_runner.py:101} INFO - Job 231: Subtask 6a_change_dir1   FutureWarning,
[2019-04-30 17:10:52,308] {base_task_runner.py:101} INFO - Job 231: Subtask 6a_change_dir1 [2019-04-30 17:10:52,308] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-30 17:10:52,447] {base_task_runner.py:101} INFO - Job 231: Subtask 6a_change_dir1 [2019-04-30 17:10:52,447] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/mixed_failing.py
[2019-04-30 17:10:52,467] {base_task_runner.py:101} INFO - Job 231: Subtask 6a_change_dir1 [2019-04-30 17:10:52,467] {cli.py:517} INFO - Running <TaskInstance: Mixed_Failing_Demo.6a_change_dir1 2019-04-30T21:10:32.665289+00:00 [running]> on host localhost.localdomain
[2019-04-30 17:10:52,473] {bash_operator.py:81} INFO - Tmp dir root location: 
 /tmp
[2019-04-30 17:10:52,473] {bash_operator.py:90} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=Mixed_Failing_Demo
AIRFLOW_CTX_TASK_ID=6a_change_dir1
AIRFLOW_CTX_EXECUTION_DATE=2019-04-30T21:10:32.665289+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2019-04-30T21:10:32.665289+00:00
[2019-04-30 17:10:52,474] {bash_operator.py:104} INFO - Temporary script location: /tmp/airflowtmpa27odd9g/6a_change_dir1nk64totf
[2019-04-30 17:10:52,474] {bash_operator.py:114} INFO - Running command: cd $AIRFLOW_HOME
[2019-04-30 17:10:52,477] {bash_operator.py:123} INFO - Output:
[2019-04-30 17:10:52,478] {bash_operator.py:131} INFO - Command exited with return code 0
[2019-04-30 17:10:56,913] {logging_mixin.py:95} INFO - [2019-04-30 17:10:56,913] {jobs.py:2562} INFO - Task exited with return code 0
